numpy/polynomial/polynomial.py

```diff
@@ -312,6 +312,12 @@ def polymulx(c):
 
     .. versionadded:: 1.5.0
 
+    Examples
+    --------
+    >>> from numpy.polynomial import polynomial as P
+    >>> c = (1,2,3)
```

Comment:
```suggestion
    >>> c = (1, 2, 3)
```
autopep8 (similar at several other places)